packages/web/components/Landing/LandingConnectButton.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
You're disabling this in multiple places. We might as well disable the rule in our eslint config 🗡️

Reply:
If there's no issue with turning it off in eslint config then yeah, cool but if i should be using a different method to get the same result, I'd like to do it the right way.

Reply:
✔️